### DISCUSSION TITLE
Compilation vs Publication? 😵

### DISCUSSION THREAD

**Discussion by Cicorione** [speedata/publisher#563] [Q&A]
Hi @pgundlach 

This is something that I've always wanted to ask you!

When I run the command `sp` and let Publisher doing its magic, what is the most appropriate terminology to describe its process?

1. Compiling?
2. Publishing?
3. Making?
4. Anything else?

Thanks  🙏

😁

**Comment by pgundlach** [comment]
Good question. I don't think it is 1, because it is more like "executing a program" and not "transfering one program representation to another one". So it is more like 2 or 3. 

4 could be "run the speedata Publisher" or "PDF generation" "PDF rendering", but niether of the terms describe exactly what happens.

**Reply by Cicorione** [reply]
**PDF Rendering** looks like more understandable although I agreed that is not what is actually happening... :thinking:
